RELEASE-NOTES.txt

```diff
@@ -11,6 +11,7 @@
 - Fix pulling to refresh on the Processing tab sometimes will not show the up-to-date orders.
 - Edit Product > Price Settings: schedule sale is now available even when either the start or end date is not set, and the sale end date can be removed now.
 - Improved stats: fixed a crash when loading improved stats on a day when daily saving time changes.
+- Improved stats: fixed the incorrect time range on "This Week" tab when loading improved stats on a day when daily saving time changes.
```

Comment:
We need to move this under the next release 4.2

Reply:
great catch! updated in https://github.com/woocommerce/woocommerce-ios/pull/2098/commits/7597543c41aa7818afdbc62f79ff5ced3f5906d5 🙂